JavascriptWebAppSample/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>VSTS Autentication ADAL.JS Sample</title>
+        <meta charset="utf-8" />
+        <style type="text/css">body { font-family: Tahoma; padding: 3em; } </style>
+        <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.11/js/adal.min.js"></script>
+    </head>
+    <body>
+        <p>
+            VSTS Authentication - ADAL.JS Sample
+        </p>
+        <p>
+            <a href="#" onclick="loginButtonPress();">
+                <div id="loginButton" class="nav"> Log in</div>
+            </a>
+            <a href="#" onclick="logoutButtonPress();">
+                <div id="logoutButton" class="nav" style="display:none;"> Log out</div>
+            </a>
+        </p>
+        <p id="username"></p>
+        <pre id="api_response"></pre>
+
+        <script type="text/javascript">
+            //========= Input Vars ================================
+            var clientId = 'xxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx';                                //Update with your app registration's Application ID (step 2.6)
+            var replyUri = 'http://localhost:8080';                                            //Where you will receive the token. (step 2.5)
```

Comment:
switched to 8081, which will be the defualt served port for http servers when TFS is using 8080.